kitsune/wiki/tests/test_views.py

```diff
@@ -2320,3 +2320,30 @@ def test_custom_locale_mapping_for_header_locale(self):
         trans_content = "This article is translated into fr"
         assert en_content not in doc_content
         assert trans_content in doc_content
+
+
+class PocketArticleTests(TestCaseBase):
+    """Pocket article redirect tests."""
+
+    def setUp(self):
+        ProductFactory()
+
+    def test_pocket_redirect_to_kb_for_document_that_exists(self):
+        """Are we redirecting to our article page?"""
+        doc = DocumentFactory(title="an article title", slug="article-title")
+        RevisionFactory(document=doc, is_approved=True)
+        pocket_style_url = f"/kb/pocket/1111-{doc.slug}"
+        response = self.client.get(pocket_style_url, follow=True)
+        self.assertContains(response, doc.title)
+
+    def test_pocket_redirect_when_kb_article_doesnt_exist(self):
+        """No match found, we should be sent to /products/pocket"""
+        pocket_style_url = "/kb/pocket/1111-wont-match"
+        response = self.client.get(pocket_style_url, follow=True)
+        path_found = False
+        for resp in response.redirect_chain:
```

Comment:
Let's just follow the redirects and check whether the final one is `products/pocket`?

Reply:
Ah, I have over-complicated this.  Changing.